.github/workflows/test-salt-linux.yml

```diff
@@ -39,7 +39,8 @@ jobs:
       image: >-
         ${{ fromJson('{
           7: "quay.io/centos/centos:centos7",
-          8: "quay.io/centos/centos:stream8"
+          8: "hub.docker.com/redhat:ubi8",
+          9: "quay.io/centos/centos:stream9"
```

Comment:
I think we can just use redhat:ubi9 here also...?

Reply:
switched